### ISSUE TITLE
Installation is not supported for this architecture: arm64

### ISSUE THREAD

**Issue by sadhasivam** [ayoisaiah/f2#19]
I am using a Mac Silicon Mx chip.  I couldn't install F2 please help
```
$ npm i @ayoisaiah/f2 -g                                                                                                                                                                                                                         ~
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated tar@2.2.2: This version of tar is no longer supported, and will not receive security updates. Please upgrade asap.
npm ERR! code 1
npm ERR! path /opt/homebrew/lib/node_modules/@ayoisaiah/f2
npm ERR! command failed
npm ERR! command sh -c go-npm install
npm ERR! Installation is not supported for this architecture: arm64
npm ERR! Invalid inputs

npm ERR! A complete log of this run can be found in:
```

**Comment by ayoisaiah** [comment]
Thank you for reporting this. I'll fix this when I make the next release.

In the meantime, could you download the darwin-arm64 binary here: https://github.com/ayoisaiah/f2/releases

**Comment by ayoisaiah** [comment]
This issue should be fixed now, try updating to v1.8.0. Feel free to reopen if it's not.
